assets/src/apps/authoring/components/PropertyEditor/custom/CustomTextbox.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+
+interface CustomTextboxProps {
+  label: string;
+  id: any;
```

Comment:
```suggestion
  id: string;
```

can this be a string? it seems it's used for the dom attr